### PR TITLE
Add turtleFSI package

### PR DESCRIPTION
**Pull request by aslakbergersen**
Checklist

- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by albansouche** [comment]
Hi, 
I'm willing to be a maintainer of the conda-forge recipe for turtleFSI.

**Comment by aslakbergersen** [comment]
@conda-forge/help-python ready for review

File: recipes/turtleFSI/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/KVSLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 84adf1e4e7a927ee43bd8a18cab75dd08d4d5f21f4b98cac7b920bbe6525bfb9
+
+build:
+  number: 0
+  skip:
+    - True  # [win]
```
Comment (dopplershift) [inline recipes/turtleFSI/meta.yaml]: How about instead:
```
- True  # [win or py2k]
```

Comment (aslakbergersen) [reply recipes/turtleFSI/meta.yaml]: Thanks for your comment @dopplershift, I have now changed it to your suggestion.

File: recipes/turtleFSI/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KVSLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 84adf1e4e7a927ee43bd8a18cab75dd08d4d5f21f4b98cac7b920bbe6525bfb9
+
+build:
+  number: 0
+  skip:
```
Comment (xhochy) [inline recipes/turtleFSI/meta.yaml]: This must be in a single line to correctly work
```suggestion
  skip: true  # [win or py2k]
```

File: recipes/turtleFSI/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    - True  # [win or py2k]
+  script: "{{ PYTHON }} setup.py install"
```
Comment (xhochy) [inline recipes/turtleFSI/meta.yaml]: Why not use pip?

File: recipes/turtleFSI/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+requirements:
+  build:
```
Comment (xhochy) [inline recipes/turtleFSI/meta.yaml]: Please remove the `build:` section completely. This should not be required.

File: recipes/turtleFSI/meta.yaml (hunk 2 above)
Comment (xhochy) [reply recipes/turtleFSI/meta.yaml]: As this doesn't have compiled code, the better practice would be to make this a `noarch: python` package without the `skip` and introduce the Python version constraint in `host` and `run`. If this is failing on Windows with missing dependencies in staged-recipes, this is ok. On the feedstock, we will build on Linux only.

File: recipes/turtleFSI/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  host:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - fenics
+    - numpy
+
+test:
```
Comment (xhochy) [inline recipes/turtleFSI/meta.yaml]: Can you add an import test?

File: recipes/turtleFSI/meta.yaml (hunk 2 above)
Comment (aslakbergersen) [reply recipes/turtleFSI/meta.yaml]: Thank you for your comment, I did not know that it was ok to have missing dependencies for a platform or only support py3 when using `noarch:python`. This is now changed.

File: recipes/turtleFSI/meta.yaml (hunk 3 above)
Comment (aslakbergersen) [reply recipes/turtleFSI/meta.yaml]: No particular reason, this is now changed to pip.

File: recipes/turtleFSI/meta.yaml (hunk 4 above)
Comment (aslakbergersen) [reply recipes/turtleFSI/meta.yaml]: Thank you for your comment, I have now removed the requirements: build section.

File: recipes/turtleFSI/meta.yaml (hunk 5 above)
Comment (aslakbergersen) [reply recipes/turtleFSI/meta.yaml]: Yes, I have added an import test.

File: recipes/turtleFSI/meta.yaml (hunk 2 above)
Comment (aslakbergersen) [reply recipes/turtleFSI/meta.yaml]: @xhochy When setting `noarch:python` the build on windows tests fail, however, this is expected behavior. Is there something I should do to make these tests pass?

Comment (xhochy) [reply recipes/turtleFSI/meta.yaml]: If it fails due to missing dependencies, then this is ok here.

**Pull request merged**